FirebaseAppDistribution/Sources/FIRAppDistributionAuthPersistence.m

```diff
@@ -11,20 +11,30 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+#import <GoogleUtilities/GULKeychainUtils.h>
 #import "FIRAppDistributionAuthPersistence+Private.h"
+#import "FIRAppDistributionKeychainUtility+Private.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 NSString *const kFIRAppDistributionAuthPersistenceErrorDomain =
     @"com.firebase.app_distribution.auth_persistence";
 
+static NSString *const kFIRAppDistributionAuthPersistenceErrorKeychainId =
+    @"com.firebase.app_distribution.keychain_id";
+static NSString *const kFIRAppDistributionAuthPersistenceServiceId = @"fire-fad-auth";
```

Comment:
FYI: We've discovered an interesting MacOS specific detail related to storing Keychain items from different applications under the same service which we had to fix with #5603. If you have plans to add Mac OS or Catalyst support you may consider using a service ID which is unique per app (e.g. contains bundleID or Firebase App ID). It will help to avoid issues in the future.

Reply:
we don't have plans to support MacOS but might be worth making it unique anyway. Will update this.

Reply:
Done